lib/auth/scram.js

```diff
@@ -124,6 +155,25 @@ ScramSHA1.prototype.auth = function(server, connections, db, username, password,
   var numberOfValidConnections = 0;
   var errorObject = null;
 
+  let cryptoMethod = this.cryptoMethod;
```

Comment:
wow I thought this was unused, but this method is like hundreds of lines long (!)